src/mrsimulator/signal_processing/apodization.py

```diff
@@ -145,3 +177,191 @@ def str_to_quantity(cls, v, values):
     def fn(self, x):
         x = self.get_coordinates_in_units(x, unit=1.0 / self.property_units["FWHM"])
         return 1.0 if self.FWHM == 0.0 else np.exp(-self.FWHM * np.pi * np.abs(x))
+
+
+class SkewedGaussian(Apodization):
+    r"""Apodize dependent variable objects of the CSDM data with a skewed Gaussian function.
+
+    The apodization function is derived from the skewed Gaussian distribution
+
+    .. math::
+        f(x) = 2*\phi(x)*\Phi(\alpha x),
```

Comment:
The ..math:: renders latex equation. You do not need `*` operator between variables. Make sure to look at the final docs. Some comments.

- remove `*`
- wrap texts like Erfi, skew in latex \text{}
- user `^` for superscript not `**`

The doc-string should be user readable. It doesn't have to be in python syntax.